db/migrations/versions/4cf5c9a457d1_.py

```diff
@@ -0,0 +1,41 @@
+"""empty message
+
+Revision ID: 4cf5c9a457d1
+Revises: 60775d659a5e
+Create Date: 2022-07-19 15:21:00.387864
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "4cf5c9a457d1"
+down_revision = "60775d659a5e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "criteria",
+        sa.Column(
+            "id",
+            sqlalchemy_utils.types.uuid.UUIDType(binary=False),
+            nullable=False,
+        ),
+        sa.Column("name", sa.Text(), nullable=True),
```

Comment:
Sql should be in single quotes: https://www.askpython.com/python/string/difference-between-single-and-double-quotes-in-python#:~:text=Single%20quotes%20are%20used%20for%20regular%20expressions%2C%20dict%20keys%20or%20SQL.

This applies to other sections of the same file.

Reply:
This is part of the `black` packages formatting sadly.